### PR TITLE
Update mainnet dev cluster config

### PR DESCRIPTION
**Pull request by adambabik**
The nodes in mainnet dev cluster changed a bit.

### CODE REVIEW

File: static/config/cluster.json [hunk 1]
```diff
@@ -34,25 +34,16 @@
...
-      "bootnodes": [
-        "enode://71bb01b58165e3262aea2d3b06dbf9abb8d5512d96e5000e7e41ab2138b47be685935d3eb119fc25e1413db00d8db231fd9d59555a1cd75229821559b6a4eb51@51.15.85.243:30303",
-        "enode://7afd119c549a7ab02b3f7bd77ef3490b6d660d5c49d0734a0c8bb23195ced4ace0bf5cde673cd5cfd07dd8d759277f3d8408eb73dc3c217bbe00f0027d06eee9@51.15.85.243:30305",
-        "enode://da8af0869e4e8047f21c1ac016b94a7b7d8e935dddd28d4272f88a1ceaee7c15e7deec9b6fd195ed3bc43748893111ebf2b2479ff44a8025ab8d598f3c97b589@51.15.85.243:30307",
-
-        "enode://80ce1b1fd0473d1acf2c94d0c852ea6a703a4a4538aa2e39834303995d1dec032b71a2f3368cea0043145a84f8f98a1c7fe99484707ede671d23eb27b7792524@163.172.176.22:30303",
-        "enode://8aa650ae7145ce65915c17ab0d369a9fcf3b16bc932a603d1bf9edf782010b54a0a069bbc607a3cf32e644276882c0d5b9b2d7e7bda7c7a10e635515746a6aa4@163.172.176.22:30305",
-        "enode://277b7896b63baae5f0e31575df5cd4912a554e50af33c0abecb95acd56000f0f69aeff2569db20b411fcb53cfd39f0eade27b8c57006b094b4d5c3091ed13458@163.172.176.22:30307"
-      ]
+      "bootnodes": []
```
Comment (themue) [inline static/config/cluster.json]: `"bootnodes"` are `"staticnodes"` now.

**Review by themue — CHANGES_REQUESTED** [review]
Renaming of field needed. What is the motivation behind the new order and why especially this one:

- 163... and 51... port 30303
- 163... and 51... port 30503
- 163... and 51... port 30403

Maybe my eyes don't catch the real sorting. 😁

**Comment by adambabik** [comment]
@themue I fixed that. Sorted by IP and then by port.

**Review by themue — APPROVED** [review]
LGTM

**Pull request merged**